www/login.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html>
+<head>
+	<meta http-equiv="Content-type" content="text/html;charset=UTF-8">
+	<title>Topomath Demo</title>
+
+	<link rel="stylesheet" type="text/css" href="css/login-base.css">
+	<link rel="stylesheet" type="text/css" href="css/login-form.css">
+
+	<script type="text/javascript">
+
+	function validateForm(){
+		var form = document.forms["problem-select"];
+		var x = form.u.value;
+		if (x==null || x==""){
+			alert("Please enter a username.");
+			return false;
+		}      
+		// If group field is empty, remove from form.
```

Comment:
This comment refers to the old group field as well, and uses "g" as the variable in line 22.  I think that  needs to be "f" instead.  Replace these references to "group" with "folder" (including the comments) and replace "g" with "f" (including in the console log.)